From 8f3e6b3ba3c08140b6891303c27ffc13a259644c Mon Sep 17 00:00:00 2001
From: Fugang Duan <fugang.duan@nxp.com>
Date: Fri, 23 Feb 2018 11:32:27 +0800
Subject: [PATCH 3468/5242] MLK-17737 net: fec: fix the struct define issue

commit  1466c27cbe38a9036a70a2085fbaf151df71a48d from
https://source.codeaurora.org/external/imx/linux-imx.git

Fix the cherry pick and merge issue by below commit on kernel 4.9:
Fixes: 19b76fd012ce ("net: fec: add stop mode support for dts register set")

Reviewed-by: Gao Pan <pandy.gao@nxp.com>
Signed-off-by: Fugang Duan <B38611@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/net/ethernet/freescale/fec.h |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/net/ethernet/freescale/fec.h b/drivers/net/ethernet/freescale/fec.h
index 1bf27e0..a6f6592 100644
--- a/drivers/net/ethernet/freescale/fec.h
+++ b/drivers/net/ethernet/freescale/fec.h
@@ -637,9 +637,9 @@ struct fec_enet_private {
 	int pps_enable;
 	unsigned int next_counter;
 
-	u64 ethtool_stats[0];
-
 	struct fec_enet_stop_mode gpr;
+
+	u64 ethtool_stats[0];
 };
 
 void fec_ptp_init(struct platform_device *pdev, int irq_idx);
-- 
1.7.9.5

